extensions/mssql/src/mssql.d.ts

```diff
@@ -544,6 +543,7 @@ export interface SqlMigrationAssessmentResultItem {
 	issueCategory: string;
 	databaseName: string;
 	impactedObjects: SqlMigrationImpactedObjectInfo[];
+	databaseRestoreFails: boolean;
```

Comment:
This seems odd to have as a boolean - the name implies to me that it's a count. Is this just coming from the nuget package itself directly or something?

Reply:
This is coming straight from the nuget package.